README.md

```diff
@@ -44,8 +44,70 @@ The keypairs are RSA with 2048 bit private keys.
       signed with the "tpm" key. A UKI signed with the "tpm" key permit access to
       the TPM password for administration.
 
+# Entities / Ownership
+**Michael** (think microsoft) owns the firmware.
+  * owns keys: uefi-pk, uefi-kek
+  * public keys for uefi-pk and uefi-kek are the UEFI Platform key and UEFI Key Exchange Key respectively.
+  * In secure boot mode, the firmware measures into PCR-7 the signer of an EFI application.
+
+**Mikey** owns a key that is in the system's UEFI DB.
+  * owns keys: uefi-db
+  * uefi-db here is a single key that is present in the uefi-db.
+  * EFI applications signed by uefi-db will be executed by the UEFI in SecureBoot mode.
+  * Mikey signs Steven's shim with Mikey's uefi-db key.
+
+**Steven** owns the shim.efi application:
+  * owns keys: uki-limited, uki-production, uki-tpm
+  * it is assumed that Steven has sole access to those 3 keys.
+  * Steven's 3 public keys are present in the shim's built-in DB.
+  * The shim's primary purpose is to validate the kernel that it will execute, ensuring that it is signed by one of the keys in it's DB.
+  * The shim measures the signer of the efi application (kernel) into PCR-7
+
+**Arthur** is the Manifest Certificate Authority.
+ * owns keys: manifest-ca
+ * As the Manifest Certificate Authority (manifest-ca), Arthur signs Oliver's Certificate Signing Request (manifest).
+
+**Katherine** owns the kernel.
+  * owns keys: uki-limited, uki-production, uki-tpm.
+  * Katherine builds and signs "Universal Kernel Initrd" (UKI) EFI applications.
+  * Katherine signs a UKI with one of uki-limited, uki-production or uki-tpm key.
+  * Katherine includes inside her signed UKI:
+    * a copy of Arthur's public manifest-ca into /manifest-ca.pem.
+    * A 'mos' binary
+
+Oliver owns the operating system
+  * owns keys: manifest, sudi-ca
+  * Oliver CSR with a specific UUID was signed by Arthur.  He owns that UUID prefix.
+  * Oliver generates and signs SUDI certificates for systems.  Those certificates are provisioned to a single machine during provisioning.
+  * Oliver uses his manifest key to sign Manifests (a root filesystem and list of containers).
+
+**Thomas** owns the tpm.
+  * owns keys: tpmpol-luks tpmpol-admin
+  * During provisioning (factory install):
+    * the tpm admin (master password) is set and configured to allow access to via policies that are signed with tpm-admin.
+    * a luks key (randomly generated long key) is generated and stored in the tpm.  Access is granted to policies signed by the tpmpol-luks key.
+    * a SUDI key is registered with the TPM.
+  * Thomas signs policies that allow access to the tpm if Katherines UKI are booted.
```

Comment:
Only if Katherne's uki-production (for lukw) or uki-tpm (for password) UKIs are booted.

Reply:
+1.